src/lib/searchHelpers.js

```diff
@@ -181,6 +203,30 @@ export function parseQueryString(queryString) {
   });
 }
 
+//adds a '-' to the beginning of a query parameter to indicate that it should be excluded
+export function markAsExcluded(queryObject, value){
+  let qp = JSON.parse(JSON.stringify(queryObject)) || {};
```

Comment:
Is this here for deep cloning `queryObject`? Elsewhere in this file the lodash `clone` function is used for that purpose. Maybe that could be used here as well?

Reply:
Yep, this is for deep cloning. I should have left a comment indicating that.